src/main/java/org/fogbowcloud/arrebol/core/ArrebolController.java

```diff
@@ -66,6 +77,12 @@ public String stopJob(Job job) {
         return job.getId();
     }
 
+    public Job getJob(String id){
+        LOGGER.debug("Acessing database to getting a job");
```

Comment:
this logline should be moved to the proper jobDAO object. Also, log the ID

Reply:
Done.